transformer/dockerfilegenerator/dotnetcoredockerfilegenerator.go

```diff
@@ -386,9 +386,9 @@ func (t *DotNetCoreDockerfileGenerator) Transform(newArtifacts []transformertype
 					logrus.Errorf("Unable to read the launchSettings.json file: %s", err)
 					continue
 				}
-				if _, ok := launchSettings.Profiles[projectName]; ok {
-					if launchSettings.Profiles[projectName].ApplicationURL != "" {
-						Urls := strings.Split(launchSettings.Profiles[projectName].ApplicationURL, ";")
+				if _, ok := launchSettings.Profiles[dotNetCoreTemplateConfig.AppName]; ok {
+					if launchSettings.Profiles[dotNetCoreTemplateConfig.AppName].ApplicationURL != "" {
+						Urls := strings.Split(launchSettings.Profiles[dotNetCoreTemplateConfig.AppName].ApplicationURL, ";")
```

Comment:
Create a new variable say, dotnetServiceName to access here. Use template config only for templates.

Reply:
As the dotnet service name (`appName`) is already decided in the detect phase, so I have named the variable `dotnetProjectName`.